Source/Core/DolphinQt2/Translator.h

```diff
@@ -0,0 +1,262 @@
+// Copyright 2015 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <QString>
+#include <QTranslator>
+
+class Translator final : public QTranslator
+{
+public:
+	QString translate(const char* context, const char* source_text, const char* disambiguation = 0, int n = -1) const override;
+	static void SetLanguage(int languageID);
```

Comment:
These should probably take a Language instead of an int, since we use another language enum elsewhere that isn't compatible.